packages/odyssey-react-mui/src/FieldComponentProps.ts

```diff
@@ -14,7 +14,7 @@ export type FieldComponentProps = {
   /**
    * If `error` is not undefined, the `input` will indicate an error.
    */
-  errorMessage?: string;
+  errorMessage?: string | string[];
```

Comment:
type union can be considered as additive change here

Reply:
I don't like double-types (array and single value) if I can avoid it.

What if we changed this so `errorMessage` is the title or "intro" and `errorMessagesList` is the bulleted list? Should simplify the logic significantly and make it more clear to other devs. `errorMessage` is always the title or the main error, and the list of them are descriptions of the error.

I think this should help with screen readers as well.